tgui/bin/tgui

```diff
@@ -29,6 +29,8 @@ base_dir="$(tgui-realpath "${base_dir}")"
 ## Make use of nvm if it exists
 if [[ -e "${HOME}/.nvm/nvm.sh" ]]; then
   source "${HOME}/.nvm/nvm.sh"
+else
+  alias node="'${base_dir}/../tools/bootstrap/node'"
```

Comment:
revert this, and add node to PATH instead

Reply:
`tools/bootstrap/node` (the #!/bin/sh version) uses pre-installed Node by default and only installs its own Node if you explicitly request it

Reply:
Have you even tested running tgui/bin/tgui on a clean repo? I guess not.

Reply:
Agreed, wrong solution owing to making a mistake elsewhere. Fixed